packages/v4/src/content/design-guidelines/components/slider/slider.md

```diff
@@ -0,0 +1,97 @@
+* Determine an increment size relative to your specific value range. Scale your increment size to accommodate values users can input in context. For example: If your range is from 0-10, structure your slider with increments of 1. If your range is from 0-100, structure your slider with increments of 1 or 10 or 25. Communicate different increments by adding tick marks to a slider.
+ 
+## Variations
+ 
+### Continuous slider
+<img src="./img/slider-continuous.png" alt="Visual of continuous slider" width="600"/>
+ 
+Allows users to select any value within the slider's range using a thumb indicator. Use this variation when any value within your slider’s range is valid for your use case.
+ 
+### Discrete slider
+ 
+<img src="./img/slider-discrete.png" alt="Visual of discrete slider" width="600"/>
+Allows users to select a value by adjusting a thumb indicator to predefined values represented by tick marks on a track. Users can only select values indicated by a tick mark.  If they attempt to select an unmarked value, the thumb indicator will always shift to the closest tick mark.
+ 
+### Slider within input groups
+ 
+When a slider is combined with other elements, it allows users to better control value selection.
+This variation combines multiple input types in one control and presents more options on how to adjust a selected value.
+ 
+For more information about using input groups, visit our [input group guidelines](/components/input-group/design-guidelines)
+ 
+#### With value input
+ 
+##### Default value input
+<img src="./img/Input-value.png" alt="Visual of slider with input value" width="600"/>
+ 
+1. **Number input in hover state**: Allows users to type in their numeric input or to adapt the slider value by using caret buttons.
+2. **Unit (optional)**: Provides a unit symbol for the value in the input field, when applicable.
+ 
+Value input is restrained by prescribed maximum and minimum parameters, and appears on the right side of the slider.
+ 
+##### Thumb value
+<img src="./img/thumb value.png" alt="Visual of  slider with thumb value" width="600"/>
+ 
+1. **Input field**: Allows users to type in their numeric input.
+2. **Unit (optional)**: Provides a unit symbol for the value in the input field, when applicable.
+ 
+In this slider variation, the value input is located over the thumb.
+ 
+For more information about using number inputs, visit our [number input guidelines](/components/number-input/design-guidelines).
+ 
+#### With action
+ 
+* **Buttons**
+ 
+<img src="./img/Slider-with-buttons.png" alt="Visual of slider with buttons" width="500"/>
+ 
+ 
+Use buttons on either side of the slider to increase or decrease selected value by set increments.
+ 
+*  **Lock feature**
+ 
+**Unlocked value**
+<img src="./img/Unlocked.png" alt="Visual of an unlocked slider" width="500"/>
+ 
+**Locked value**
+<img src="./img/Locked.png" alt="Visual of a locked slider" width="500
```

Comment:
Same thing here, need to add a line of space between the header and image

Reply:
Can you also try to indent the unlock and locked things to be indented underneath the "lock feature" bullet point? Cause right now the formatting looks a bit odd

Reply:
@mmenestr what about now?

Reply:
It's not indenting - maybe for the unlocked value and locked value don't do the hashtag thing, and just bold it intead and that's it

Reply:
I'm trying to figure out why the unlocked and locked value are showing up so big and bolded still... even though I see here you didn't make it a header? It's also still not indenting hmmm

![image](https://user-images.githubusercontent.com/51165119/126644255-41eea76c-a4d4-489d-b82b-847baf97a16c.png)